include/cocaine/context.hpp

```diff
@@ -152,6 +169,9 @@ class context_t {
     // because services are allowed to start and stop other services during their lifetime.
     synchronized<service_list_t> m_services;
 
+    //Flag set after initialization
+    bool ready;
```

Comment:
Is it possible to add a new behavior to `boost::signals2` so that the slot could be blocked in a deferred sort of state where it accumulates all the incoming signals but triggers them only when it's unblocked again?